### PR TITLE
Tweak category buttons

### LINKED ISSUE #266: tweaking new design
Although the new design elements are nice and aesthetically pleasing, they do not, in my opinion, blend in nicely with the rest of the site.
I would suggest:

1. remove the lines around the section names. 
2. change the magnifying glass icon to something less search-related? Maybe a three-dot icon. Or maybe we don't need it at all?

An alternative would be to let someone play around with the different options and suggest changes. 

What do you think?

Without the solid border, it will look like that:
**Additional context**
Screenshot from 2022-10-18 18-58-27

### PR DESCRIPTION
**Pull request by dokterbob**
Please read full commit messages for background/rationale.

Closes #266.

Would be great if we could throw this live before the camp. ^^

### CODE REVIEW

**Comment by femans** [comment]
I like it but i think it should only be visible on the All types page, right? No need to display it anymore after you click on it. N.b. Just quickly checked on my phone, will look better later.

File: src/components/searchViewComponents/BaseList.vue [hunk 1]
```diff
@@ -179,6 +176,12 @@ const queryPage = computed({
       </v-row>
     </slot>
 
+    <v-row v-if="hasResults" justify="center" class="mt-2 mb-1">
```
[the remark this answers is not in the document]
Comment (femans) [reply src/components/searchViewComponents/BaseList.vue:179]: Also needs to fix responsiveness. with mdAndUp, it is now a small little pill, but on smAndDown it is full width.

[the remark this answers is not in the document]
Comment (dokterbob) [reply src/components/searchViewComponents/BaseList.vue:179]: > needs: `v-if="anyFileType"

Resolved in 167d54cd93a78946a0ed69fe6137f113fa167739.

> Also needs to fix responsiveness. with mdAndUp, it is now a small little pill, but on smAndDown it is full width.
This is intended behaviour: I wanted to visually discriminate the 'Load more' intent from the 'Show detail' intents whenever screen space allows it. On smaller screens the shape of the 'pill' starts to look weird (it becomes some kind of confused oval) and the difference in size is reduced so I figured it makes sense to make it full with. For horizontal view on mobiles there is the in-between solution. Feel free to try this look 'n feel in a browser window and/or in various 
user agents (like I did).

File: src/components/searchViewComponents/BaseList.vue [hunk 2]
```diff
@@ -114,26 +120,17 @@ const queryPage = computed({
...
-        :to="{ ...route, query: { ...route.query, type: fileType } }"
-        class="flex-grow-1"
-      >
-        <div
-          class="w-100 d-flex justify-start align-center text-ipfsPrimary-lighten-1 v-btn v-btn--density-default v-btn--size-default v-btn--variant-outlined"
-          color="ipfsPrimary"
+      <hyperlink :disabled="!anyFileType" :to="viewAllTo" class="flex-grow-1">
+        <!-- Note: v-btn has a "to" prop as well, which should eliminate the need for hyperlink here. However, it causes the btn to be rendered as 'tonal', overriding the text variant here -->
+        <v-btn
+          class="categoryBtn"
```
[the remark this answers is not in the document]
Comment (dokterbob) [reply src/components/searchViewComponents/BaseList.vue:126]: Resolved in 0977df41455e913bb9daa73f670805adee8886ef

File: src/components/searchViewComponents/BaseList.vue [hunk 3]
```diff
@@ -114,26 +120,17 @@ const queryPage = computed({
...
-        :disabled="!anyFileType"
-        :to="{ ...route, query: { ...route.query, type: fileType } }"
-        class="flex-grow-1"
-      >
-        <div
-          class="w-100 d-flex justify-start align-center text-ipfsPrimary-lighten-1 v-btn v-btn--density-default v-btn--size-default v-btn--variant-outlined"
-          color="ipfsPrimary"
+      <hyperlink :disabled="!anyFileType" :to="viewAllTo" class="flex-grow-1">
+        <!-- Note: v-btn has a "to" prop as well, which should eliminate the need for hyperlink here. However, it causes the btn to be rendered as 'tonal', overriding the text variant here -->
+        <v-btn
```
[the remark this answers is not in the document]
Comment (dokterbob) [reply src/components/searchViewComponents/BaseList.vue:125]: I removed the named class, I wasn't aware of this utility class, and added a single `style`.

For several reasons, I think it's wise to use existing components whenever possible:

1. Maintenance and testing; if we make custom components, it means more code within our responsibility, which implies more testing, more edge cases and more work in the long run.
2. Consistent design; while trying to work with your classes, I found out several subtle spacing issues (1-5 pixel domain) which (especially when aggregated) reduces the aesthetic experience.
3. Verbosity; a tag like `<v-btn>` is much clearer to read, plus it has documentation.

But if you insist I can change things back, it's just it'll be a bit more work.

File: src/components/searchViewComponents/BaseList.vue [hunk 4]
```diff
@@ -114,26 +120,17 @@ const queryPage = computed({
...
-      <hyperlink
-        :disabled="!anyFileType"
-        :to="{ ...route, query: { ...route.query, type: fileType } }"
-        class="flex-grow-1"
-      >
-        <div
-          class="w-100 d-flex justify-start align-center text-ipfsPrimary-lighten-1 v-btn v-btn--density-default v-btn--size-default v-btn--variant-outlined"
-          color="ipfsPrimary"
+      <hyperlink :disabled="!anyFileType" :to="viewAllTo" class="flex-grow-1">
+        <!-- Note: v-btn has a "to" prop as well, which should eliminate the need for hyperlink here. However, it causes the btn to be rendered as 'tonal', overriding the text variant here -->
```
[the remark this answers is not in the document]
Comment (dokterbob) [reply src/components/searchViewComponents/BaseList.vue:124]: Please suggest another way to refer to undocumented relevant behaviour of this particular component. 

It took quite a bit of experimenting to discover this and this comment is intended to prevent our future selves from having to do these experiments again (trying to optimise out the hyperlink only to discover that `to` for `<v-btn>` changes the rendering.

**Review by femans — APPROVED** [review]
